content/docs/how-we-work/security.md

```diff
@@ -10,6 +10,29 @@ Security is an everlasting concern in the world of software engineering. Underst
 
 That said, security is a wide and deep topic and a single person cannot hope to span the width and depth of this challenging area. It is the team's responsibility to understand security at various levels and implement safeguards.
 
-For example, an application relies on database credentials to connect to the database. There can be API keys using which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them.
+For example, an application relies on database credentials to connect to the database. There can be API keys using which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them. Use the environment variables or vaults for storing and retrieving them.
 
-Use the environment variables or vaults for storing and retrieving them.
+## Server and Website Security
+
+In particular, we practice server and site security by using only sanitized databases, taking care not to install restricted access files on development or personal instances outside the project defined security accreditation boundary and scrubbing unneeded data from our development systems
```

Comment:
Please break this sentence. Also, what do you mean by "security accreditation boundary"? How is a reader supposed to interpret that? There's also a missing period at the end of this sentence.

Reply:
You are right.

Reply:
As per my understanding, "project defined security accreditation boundary" are related to some specific authentication/validation/security guidelines that we must adhere to when working on a specific project.

These lines; I tried my best to use contents of [Security and Compliance](https://axelerant.atlassian.net/wiki/spaces/AH/pages/4040824622/Security+and+Compliance). Since we are collaborating here for the contents https://docs.google.com/document/d/1eA7IVZwF3kokfa8uEkNudHu6VRQrfr4-cgC59YEmHBA/edit#heading=h.soaz83frhrw6, let's finalise the content for it. My recent trial is [this](https://docs.google.com/document/d/1eA7IVZwF3kokfa8uEkNudHu6VRQrfr4-cgC59YEmHBA/edit?disco=AAAA5ZFNqSk) one please review @Yemaneberhan-Lemma .

Reply:
@axelabhay, we can argue accreditation this and boundary that. We can stay up all night trying to guess what was going on in the original writer's mind when they wrote this 😆. Or we can accept that this is just a fanciful word jumble that never made sense and will likely mean nothing to the reader (the main person we should worry about). If we do accept that, we can forget that it was ever in Confluence and make our Engineering handbook readable to humans.

Reply:
Yes, we have to accept this, our intention is to inspire our readers to read the handbook, and not specifically our minds 😃. I will collaborate with @Yemaneberhan-Lemma and will take care to present the content in simple language.